agent_docs/workflow.md

```
@@ -0,0 +1,127 @@
+
+### Commit Message Format
+
+```
+<subsystem>: <what changed> (PROJQUAY-####)
+
+<why this change was made>
+```
+
+### Branch Naming
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)